docs/templates/addon/content_type/index.rst

```diff
@@ -1,7 +1,12 @@
-=================================
-Creating a Dexterity Content Type
-=================================
-`Title <http://link>`_
+============
+content_type
+============
```

Comment:
`Content Type`, this is a header :)

Reply:
but in this case it's the name of the template, that's why i wouldn't use a different way of writing here.
I also use it to gerate the list of templates and they should have the correct names.
On other places I'll try to take this into account. thx